index.bs

```diff
@@ -7142,7 +7142,7 @@ To <dfn>deserialize local value</dfn> given |local protocol value|, |realm| and
       1. Let |pattern| be the value of the <code>pattern</code> field of |local protocol value|.
 
       1. Let |flags| be the value of the <code>flags</code> field of |local protocol value| or
-         undefined if no such a field.
+         null if no such a field.
```

Comment:
This is calling the ecma262 RegExp constructor, which takes undefined, not null, for missing flags, so this is wrong.